nni/algorithms/compression/v2/pytorch/utils/common.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
common in utils doesn't make much sense. Looks like "attr.py"

Reply:
good suggestion, rename it